docs/usage.md

```diff
@@ -0,0 +1,55 @@
+| [Home](../README.md) |
+|----------------------|
+
+# Usage
+
+The **Playbook Execution Wizard** widget renders playbook execution logs and related comments on the wizard for a more fine-grained control on your processes.
+
+This wizard can currently with used in tandem with following widgets:
+
+1. JSON to Grid Widget
```

Comment:
Can we give doc hyperlink of these widget also.
@cs-abhishek-shukla

Reply:
Docs are in separate repositories and hence linking may break the doc flow